packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-payment-strategy.spec.ts

```diff
@@ -35,6 +35,7 @@ describe('PayPalCommercePaymentStrategy', () => {
     let paypalCommerceIntegrationService: PayPalCommerceIntegrationService;
     let paypalSdk: PayPalSDK;
     let strategy: PayPalCommercePaymentStrategy;
+    let restartMock: any;
```

Comment:
I don't think this variable should be defined on global level since it used only in one place. You can move it to test level and mock it there.

Also, we should not use 'any' type in our project

Reply:
fixed